mutt/list.h

```diff
@@ -57,5 +58,7 @@ struct ListNode *mutt_list_insert_head (struct ListHead *h, char *s);
 struct ListNode *mutt_list_insert_tail (struct ListHead *h, char *s);
 bool             mutt_list_match       (const char *s, struct ListHead *h);
 size_t           mutt_list_str_split   (struct ListHead *head, const char *src, char sep);
+void             mutt_list_copy_tail   (struct ListHead *dst, const struct ListHead *src);
+size_t           mutt_list_write       (const struct ListHead *h, struct Buffer *buf);
```

Comment:
Please sort this list

Reply:
Ahh, didn't realize it was sort(1)ed.  Will do.